Feature/ground

oceull opened the pull request:
first merger from lefthanded luc

Commit:
Create ground materials with 3 different highlights

oceull committed:
fade in at start of level

oceull committed:
Selector of tile ready

oceull committed:
first level design

oceull committed:
made the highligh darker
made the selection move up a bit

oceull committed:
fall down at start of level

oceull committed:
hahaha

oceull committed:
remove some in mainscene

oceull committed:
final

Jop-van-Roosmalen committed:
merged develop into ground

Duitel commented on Assets/Materials/BikePath.mat:
```diff
@@ -0,0 +1,81 @@
+%YAML 1.1
```
Misschien alle materialen voor blokjes in een submap onder materials?

Duitel commented on Assets/Scripts/BoardSelect.cs:
```diff
@@ -0,0 +1,198 @@
+public class BoardSelect : MonoBehaviour
+{
+    [SerializeField] private GameObject[] _groundGameObject;
+    private int height; // tp the right
```
tp?

oceull replied:
ik weet niet hpe je die niet kunt begijpen. Btw het pntbijt was lekker

Duitel commented on Assets/Scripts/BoardSelect.cs:
```diff
@@ -0,0 +1,198 @@
+    private int height; // tp the right
+    private int width; // to the left
+
+    private int[,] _floorplan =
```
wordt deze nog ergens uitgelegd, anders misschien hier nog even een comment?

oceull replied:
phja

Duitel commented on Assets/Scripts/BoardSelect.cs:
```diff
@@ -0,0 +1,198 @@
+    }
+
+    // Update is called once per frame
+    void FixedUpdate()
```
Is dit om het minder snel te laten bewegen?

oceull replied:
si

Duitel commented on Assets/Scripts/Ground.cs:
```diff
@@ -0,0 +1,83 @@
+    void Start()
+    {
+        _renderer = GetComponent<Renderer>();
+        _thisTransform = this.GetComponent<Transform>();
```
is dit `this.GetComponent<Transform>()` niet hetzelfde als `transform`

oceull replied:
yes

Duitel commented on Assets/Scripts/Ground.cs:
```diff
@@ -0,0 +1,83 @@
+        _renderer = GetComponent<Renderer>();
+        _thisTransform = this.GetComponent<Transform>();
+        _thisTransform.position = new Vector3(_thisTransform.position.x, -0.5f, _thisTransform.position.z);
+        _position = _thisTransform.position;
```
_position nodig als globale variabele? Anders liever alleen lokaal maken aan deze methode voor de leesbaarheid en betrouwbaarheid

Duitel replied:
ik zou gewoon _position in transform.position veranderen

oceull replied:
_position wordt gebruikt in de laatste functie BeginState()

het is aangepast!

Duitel commented on ProjectSettings/TimelineSettings.asset:
```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
```
Deze nodig? Anders weghalen

oceull committed:
Update BoardSelect.cs

oceull committed:
Delete TimelineSettings.asset

oceull committed:
Added comments in scripts boardselect

oceull committed:
added comments
interchanged: "_thisTransform = this.GetCompenent<Transform>()" for "transform"

oceull committed:
added more comments

oceull committed:
Clean-up the folder structure
